lib/zipkin-tracer/trace.rb

```diff
@@ -199,12 +162,29 @@ def to_i; @i128; end
 
   # A span may contain many annotations
   class Span
-    attr_accessor :name, :annotations, :binary_annotations, :debug
+    module Tag
+      METHOD = "http.method".freeze
+      PATH = "http.path".freeze
+      STATUS = "http.status".freeze
```

Comment:
this is wrong, it should be http.status_code though possibly was always incorrect here

Reply:
updated https://github.com/openzipkin/zipkin-ruby/pull/141/commits/d4b62540cef69d5c012ee05a2355e090339c1f56